<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditMaterial</class>
 <widget class="QDialog" name="EditMaterial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EditMaterial</string>
  </property>
  <widget class="QPushButton" name="OKBtn">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>560</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>560</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>391</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="descLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Descriptions:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="materialBehaviors">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>431</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Material Behaviors</string>
   </property>
   <widget class="QListView" name="behaviors">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>411</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="behaviorLayout"/>
   </widget>
  </widget>
  <widget class="QPushButton" name="editDescBtn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>40</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="description">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>431</width>
     <height>281</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_0">
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>431</width>
       <height>281</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_1"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
